src/NuGet.Clients/PackageManagement.VisualStudio/SolutionRestore/RestoreOperationLogger.cs

```diff
@@ -182,19 +202,16 @@ public void WriteLine(VerbosityLevel verbosity, string format, params object[] a
         {
             ThreadHelper.ThrowIfNotOnUIThread();
 
-            if (OutputVerbosity >= (int)verbosity && _buildOutputPane.Value != null)
+            if (OutputVerbosity >= (int)verbosity && _outputWindowPane != null)
```

Comment:
Constructor is private, there's no other way to create the instance.
Output window pane can be null if it's not open, for instance on solution open.